src/Utility/requests.js

```diff
@@ -82,19 +82,27 @@ export function fetchTemplate(locale, request_type, company = null, suffix = 'de
                     return response.text();
                 case 404:
                     if (locale !== REQUEST_FALLBACK_LANGUAGE) return fetchTemplate(REQUEST_FALLBACK_LANGUAGE, template);
-                    throw new CriticalException('Request template cold not be found', {
-                        locale: locale,
-                        template: template,
-                        template_url: template_url,
-                        response: response
-                    });
+                    throw new CriticalException(
+                        'Request template could not be found.',
+                        {
+                            locale: locale,
+                            template: template,
+                            template_url: template_url,
+                            response: response
+                        },
+                        'Request template could not be found.'
```

Comment:
Sorry if I didn't make this clear but user-facing content should really be translated.

Reply:
In error messages? I always feel like I don't want to involve to much of the code in the error producing…

Reply:
But that message is very prominently shown in the error modal. And previous uses are also using `t()` for the `enduser_message`.

If `t()` fails, I'm pretty sure we'll notice anyway…